component_event/tests/test_event.py

```diff
@@ -12,8 +12,9 @@
     ComponentRegistryCase,
     TransactionComponentRegistryCase,
 )
-from odoo.addons.component_event.components.event import skip_if
-from odoo.addons.component_event.core import EventWorkContext
+
+from ..components.event import skip_if
+from ..core import EventWorkContext
```

Comment:
Hi, many thanks for your work on migrating these addons!

I  disagree with changing these imports in test files though. But I know you did it because pylint-odoo complains, they are not blocking though.
Tests should be able to run on a package "from the outside", whatever the place the tests are located.
I wrote a comment on #382 that I don't find anymore.
But tests are not "part" of the python package, usually in python libs, tests are outside of the libs, and you test them using their import path. Which also proves that outside modules can import them btw.
The fact that using relative import works here is only an artifact.

Actually, I should open an issue on pytest-odoo

Reply:
Hi, you made a very good point, I think you're right. So, I just added the Python dependency that was missing.
Thanks! :-)

Reply:
On my side I'm waiting for a review on https://github.com/OCA/connector/pull/382  from initial authors to update component_event PR with this missing lib

Reply:
Ok I close this PR then.
Thanks!